apps/explorer/src/app/components/links/asset-link/asset-link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Routes } from '../../../routes/route-names';
+import { useExplorerAssetQuery } from './__generated__/Asset';
+import { Link } from 'react-router-dom';
+
+export type AssetLinkProps = {
```

Comment:
suggestion ():  to make sure no unintended prop goes in to the Link component, you can merge this with the Props of the Link, e.g:

```
import { ComponentProps } from 'react'
import { Link } from 'react-router-dom'

type AssetLinkProps = ComponentProps<typeof Link> & {
  id: string
}
```

note: react router might export the props for the link type, not sure if does, `ComponentProps` is a good way to get around not having access to certain component's props type in case.